src/engine/Settings.cs

```diff
@@ -63,10 +63,11 @@ public enum StrainBarVisibility
     // Graphics Properties
 
     /// <summary>
-    ///   Sets whether the game window is in fullscreen mode
+    ///   Sets window mode of the game window
     /// </summary>
     [JsonProperty]
-    public SettingValue<bool> FullScreen { get; private set; } = new(true);
+    public SettingValue<DisplayServer.WindowMode> DisplayMode { get; private set; } =
+        new(DisplayServer.WindowMode.Fullscreen);
```

Comment:
Also as basically everyone on Windows is impacted, I think we might just want to have exclusive fullscreen be the default mode, because we'd still get bug reports about the border and wrong resolution probably if it isn't picked by default.